txjuju/testing/api.py

```diff
@@ -75,19 +75,56 @@ def response(self, response, requestId=None):
         self._fire(payload, requestId)
 
     def responseLogin(self, endpoints=[u"host"]):
-        api_servers = [
-            [{"NetworkName": "net-%d" % index,
-              "Port": 17070,
-              "Scope": "local-cloud",
-              "Type": "ipv4",
-              "Value": endpoint}]
-            for index, endpoint in enumerate(endpoints)]
-        return self.response(
-            {"EnvironTag": "environment-uuid-123",
-             "Servers": api_servers})
+        if self.version.startswith("2."):
+            api_servers = [
+                [{"space-name": "net-%d" % index,
```

Comment:
I thought of that and didn't want to bother.  The effort of refactoring for reusability like that didn't seem worth it.